modules/n1ql/pages/n1ql-language-reference/indexing-arrays.adoc

```diff
@@ -268,17 +268,17 @@ The following examples use the {install-sample-buckets}[travel-sample] bucket th
 .Indexing all DISTINCT elements in an array
 ====
 
-.C{example-number}: Create an index on all schedules
-[[C1,C{example-number}]]
+C{counter:example-counter}: Create an index on all schedules
```

Comment:
Is there a reason why the C examples (CREATE INDEX) use plain paragraphs (no initial dot), whereas the Q examples (SELECT) use block titles (with initial dot)?

```
C{counter:example-counter}
```

versus

```
.Q{example-counter}
```

Reply:
My mistake. That's a search and replace typo. It should be the same as it was in the original. Are you able to correct it?

Reply:
No problem, I'll update the titles in a  new PR.

I need to look at this in more detail, because after this change the example counters don't seem to work:

![Screen Shot 2021-05-27 at 00 18 24](https://user-images.githubusercontent.com/39966290/119743290-77820380-be81-11eb-9416-d77af60e6567.png)

(Created with Antora 3.0.0 alpha 2)